crates/rpc/src/rpc_models.rs

```diff
@@ -118,4 +118,5 @@ pub struct Storefront {
     pub description: String,
     pub favicon_url: String,
     pub logo_url: String,
+    pub created_at: Option<NaiveDateTime>,
```

Comment:
```suggestion
    pub created_at: Option<Timestamp>,
```

The `rpc_models` module uses a `Timestamp` struct to enforce consistent formatting of `NaiveDateTime` objects.